include/oneapi/dpl/pstl/pstl_config.h

```diff
@@ -169,4 +177,11 @@
 // broken macros
 #define _PSTL_ICC_18_OMP_SIMD_BROKEN (__INTEL_COMPILER == 1800)
 
+// TODO: enable when fixed
```

Comment:
Probably, the comment should be more informational.

Reply:
It's been clarified.